### DISCUSSION TITLE
queue.getFiltersEnabled is not a function

### DISCUSSION THREAD

**Discussion by BEAST2802** [Androz2091/discord-player#706] [Q&A]
**Describe the bug**

Don't know what happened now. Before i was using everything in index.js and it was working perfectly but after i transferred to handlers it is returning that error. I used ctk's handler
**To Reproduce**

**Expected behavior**

**Screenshots**

**Please complete the following information:**
 - Node Version: [16.6.x]
 - Discord Player Version: latest
 - Discord.js Version: 13.1.0
![Screenshot_2021-08-16-15-02-16-61](https://user-images.githubusercontent.com/82759519/129545188-9e10ca7c-ce1a-494e-8697-0e3a56949bc0.jpg)

**Additional context**

**Comment by BEAST2802** [comment]
Please help me. No one was helping in discord so i opened in github

**Comment by twlite** [comment]
Something's wrong on your side.  Maybe re-check your handler, verify your parameters?

**Comment by BEAST2802** [comment]
Hmm tho i checked everything but no errors was found. Let me send the handler code that i used

**Comment by BEAST2802** [comment]
```js
const { readdirSync } = require("fs");
const ascii = require("ascii-table");


// Create a new Ascii table
let table = new ascii("Player Events");
table.setHeading("Events", "Load status");

module.exports = (client) => {

  const commands = readdirSync(__dirname.replace("\handlers", "\player")).filter(file => file.endsWith(".js"));

  for (let file of commands) {

    try {
    let pull = require(`${__dirname.replace("\handlers", "\player")}/${file}`);

    if (pull.event && typeof pull.event !== "string") {
      table.addRow(file, `❌ -> Property event should be string.`);
      continue;
    }

    pull.event = pull.event || file.replace(".js", "")

    client.player.on(pull.event, pull.run.bind(null, client))

    table.addRow(pull.event, '✅');

    } catch(err) {

  console.log("Error While loading/executing command, join for help : https://withwin.in/dbd")
  console.log(err)
  table.addRow(file, `❌ -> Error while loading event, join for help :  https://withwin.in/dbd`);
    }
  }

   console.log(table.toString());
}
```

**Comment by BEAST2802** [comment]
Is there any errors in this handler which u can see?

**Comment by BEAST2802** [comment]
Please help me 🙏
